backend/lobby/consumers.py

```diff
@@ -101,8 +104,36 @@ def gamelog_send_handler(self, data):
 
         self.send_server_response_to_client(username, room, mud_gamelog_input)
 
+    def selected_card_handler(self, data):
+        username = data['user']
+        room = data['room']
+
+        print("selected card handler")
+
+        payload = data['payload']
+
+        self.send_server_response_to_client(username, room, payload)
+
+    def get_the_three_store_cards(self):
+        card_market = self.master_game_controller.deck_handler.store
+        # card_market is now a list of three random cards, modify as needed
+        card1 = card_market[0]
+        card2 = card_market[1]
+        card3 = card_market[2]
+        return card_market
+
+    # This test is temporary to prove the GameConsumer has access to the card store
+    def test_get_store_cards():
```

Comment:
can you add this to consumers_test.py?

Reply:
Moved this test to test_consumers.py file